client/src/components/Footer/Footer.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap'); /* Will eventually delete */
+
+.Footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    min-width: 100vw;
+    height: 50px;
+    background-color: #FFCB06; /* Change colors once css variables are merged */
+    font-family: 'Reem Kufi', sans-serif; /* Will eventually delete */
+}
+
+.Footer a, .Footer a:active, .Footer a:link, .Footer a:visited { 
+    color: #252525; /* Change colors once css variables are merged */
+}
+
+.footer-left, .footer-right, .footer-center {
+    display: flex;
+    text-align: center;
+    margin: 3vw;
+}
+
+.footer-left {
+    width: 20%;
+}
+
+.footer-center a {
+    text-decoration: none;
+}
+
+.footer-right {
+    justify-content: space-around;
+    width: 20%;
+}
+
+@media only screen and (max-width:1000px) {
+
+    .Footer {
+        height: 75px;
+    }
+
+    .footer-right {
+        flex-direction: column;
+        justify-content: right;
+        line-height: 20px;
+    }
+}
```

Comment:
This is a great start to responsive! On mobile can we have all three columns stack?

Reply:
@ZakkMan Made edits and pushed them up. The footer collapses to a single column below 400px. All should be ready.